### PR TITLE
[py-ipyparallel] setuptools.config.StaticModule moved

### PR DESCRIPTION
**Pull request by stephenmsachs**
... in py-setuptools@61

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-ipyparallel/package.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ class PyIpyparallel(PythonPackage):
     depends_on('py-jupyterlab@3.0:3', type='build', when='@7.1:')
     depends_on('py-packaging', type='build', when='@7.1:')
     depends_on('py-setuptools@40.8:', type='build', when='@7.1:')
+    depends_on('py-setuptools@:60', type='build', when='@:8.0.0')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ipyparallel/package.py]: Has this been fixed in 8.0.1?

Comment (stephenmsachs) [reply var/spack/repos/builtin/packages/py-ipyparallel/package.py]: Actually in 8.2.1. Let me change that.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-ipyparallel/package.py]: If it was fixed in 8.2.1 then the range should be `@:8.2.0`, version ranges are inclusive in Spack.

Comment (stephenmsachs) [reply var/spack/repos/builtin/packages/py-ipyparallel/package.py]: Thanks!..... this is embarassing 😄

**Review by adamjstewart — APPROVED** [review]

**Comment by stephenmsachs** [comment]
@spackbot run pipeline

**Pull request merged**